wsmaster/che-core-api-workspace-shared/src/main/java/org/eclipse/che/api/workspace/shared/Constants.java

```diff
@@ -68,6 +68,18 @@
    * Workspace#getAttributes}
    */
   public static final String ERROR_MESSAGE_ATTRIBUTE_NAME = "errorMessage";
+  /**
+   * Contains Workspace.Next features list that should be added to a workspace.
+   * Should be set/read from {@link Workspace#getAttributes}.
+   *
+   * <p/>Value is comma separated list of features in a format:
+   * '< feature1Name >/< feature1Version >,< feature2Name >/< feature2Version >'
```

Comment:
Why is `/` symbol chosen as a separator of name and version? Wouldn't `:` symbol more conventional?

Reply:
It is better to ask @skabashnyuk - it is Feature API that declares format.

Reply:
The format is still under discussion. I do like ```:```  Fill free to comment. https://github.com/eclipse/che/issues/9895